DEPLOYMENT.md

```diff
@@ -0,0 +1,176 @@
+## ODM2 RESTful Web Services -- Deployment Guide
```

Comment:
I think this should go in a folder called docs.

Reply:
Also, rename the file to something more specific like, `production_deployment.md`

Reply:
Done, should we move anything else to the docs folder?

Reply:
Nope.

Reply:
Thanks!